docs/source/extension/extension_migration.rst

```diff
@@ -323,17 +330,17 @@ bumped their major version (following semver convention). We want to point out p
         The renderer id must follow the convention ``<ISettingRegistry.IPlugin.id>.<propertyName>``. This is to
         ensure a custom renderer is not used for property with the same name but different schema.
 - ``@jupyterlab/translation`` from 3.x to 4.x
-   Renamed the method ``locale`` into the property ``languageCode`` in the ``NullTranslator``
+  Renamed the method ``locale`` into the property ``languageCode`` in the ``NullTranslator``
 - ``@jupyterlab/vdom`` and ``@jupyterlab/vdom-extension`` have been removed.
-   The underlying [vdom](https://github.com/nteract/vdom) Python package is unmaintained.
-   So it was decided to drop it from core packages.
+  The underlying [vdom](https://github.com/nteract/vdom) Python package is unmaintained.
```

Comment:
That's a markdown link in rst document:

![Screenshot from 2023-04-12 20-42-34](https://user-images.githubusercontent.com/5832902/231567469-501503d7-064e-4aa7-b46d-864172ab9b7b.png)

we can either use rst syntax or keep bare URL as there is some nice formatting for GitHub URLs in our docs:

```suggestion
  The underlying https://github.com/nteract/vdom Python package is unmaintained.
```

Reply:
Replace by RST syntax